boards/arm/stm32h7/stm32h747i-disco/src/stm32_sdmmc.c

```diff
@@ -0,0 +1,175 @@
+/****************************************************************************
+ * boards/arm/stm32h7/stm32h747i-disco/src/stm32_sdmmc.c
+ *
+ *   Copyright (C) 2016-2019 Gregory Nutt. All rights reserved.
+ *   Author: Gregory Nutt <gnutt@nuttx.org>
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
```

Comment:
New files should have Apache license. Please copy the text from [here](https://github.com/apache/incubator-nuttx/blob/master/sched/sched/sched_getcpu.c)

Reply:
@patacongo Could you please give your explicit approval for this one.

Reply:
No, we cannot bring any new 3rd party code into the repository. We can never do that unless they provide a software grant (SGA).

Reply:
I meant the approval to change the license to Apache.

Reply:
Ahh... talking with Abdelatif, I missunderstood the question.  I am the copyright holder and the old SDMMC code and do give my approval to change the license header to the Apache 2.0 license header cited above.

The one with the PX4 copyright must also have an Apache 2.0 header to be considered.

BTW:  The current version of my original SDMMC file does now have an Apache header and also includes requires changes that you do not have in this PR.

Reply:
license is now changed to Apache